SpatialGDK/Source/SpatialGDK/Private/Interop/ActorSystem.cpp

```diff
@@ -535,6 +560,22 @@ void ActorSystem::HandleActorAuthority(const Worker_EntityId EntityId, const Wor
 					if (Channel != nullptr && Channel->IsAutonomousProxyOnAuthority())
 					{
 						Actor->RemoteRole = ROLE_AutonomousProxy;
+
+						// Flush PC interest on handover
+						if (GetDefault<USpatialGDKSettings>()->bUseClientEntityInterestQueries && Actor->IsA<APlayerController>())
+						{
+							const Worker_EntityId ControllerEntityId =
+								NetDriver->PackageMap->GetEntityIdFromObject(Actor->GetNetConnection()->PlayerController);
```

Comment:
Just wonder if there is any difference here if casting actor to APlayerController instead of getting the player controller off  the connection?

Reply:
It shouldn't be, but we're just being safe for now - hence the following lines `ControllerEntityId == EntityId`